README.md

```diff
@@ -1,62 +1,105 @@
 
-> A website and API for getting information on students at Yale College.
+A website and API for getting information on students at Yale College.
 
 ![Screenshot](screenshot.png)
 
 ## Initial Setup
 
 To develop changes to the application, you'll need to run it locally for testing.
 
-This guide assumes, as prerequisites, that you have:
+This guide assumes, as prerequisites, that you have A MacOS or Linux-based OS (if you use Windows, you can still follow along, but some commands may be different).
 
-- A MacOS or Linux-based OS (if you use Windows, you can still follow along, but some commands may be different)
-- Access to, and basic understanding of, the terminal on your computer
-- Python 3
-  - Modern MacOS versions and most Linux distributions include Python 3 by default.
-  - If you use a version of MacOS with only Python 2, first [install Homebrew](https://brew.sh/#install), a helpful package manager for MacOS, then use it to install Python 3:
-  ```sh
-  brew install python3
-  ```
-- PostgreSQL, which is needed for working with our databases:
-  - If you're on MacOS, use Homebrew (or another package manager) to install PostgreSQL
-  ```sh
-  brew install postgresql
-  ```
+### Install Homebrew
+- If you're on a Mac, first [install Homebrew](https://brew.sh/#install).
+- If you're on Linux, follow along with your distro's package manager (`pacman`, `apt-get`, `yum`, etc).
 
-Next, make a "fork" of the `api` repository by clicking the Fork button in top right of repository page. This will make a copy of the codebase under your own personal user account that you'll be able to modify as you wish. When you make changes to your own repository, you can then send those changes in to be approved and merged into the main codebase. (Read on to learn how!)
+### Install Python
+This project requires Python version 3.10.7. You may already have Python on your machine—it comes preinstalled with macOS—but it may
+be the wrong version.
 
-Next, clone your fork of the repository to your local computer and enter the newly created directory:
+To check your python version, run:
+```bash
+python3 --version
+```
+If it says anything other than `3.10.7`, keep following along.
 
-```sh
-git clone https://github.com/[YourUsernameGoesHere]/api
-cd api
+We are going to use `pyenv` to install multiple versions of Python on our machine, and `pyenv-virtualenv` to manage dependencies in our package. Run:
+```bash
+brew install pyenv
+brew install pyenv-virtualenv
+```
+Now, we must add pyenv to our PATH. PATH is a special bash variable that tells the shell what executables we can run. Open your bash profile with:
+```
+nano ~/.bash_profile
+```
+and add the following lines at the bottom:
+```bash
+export PATH="$PYENV_ROOT/bin:$PATH"
+eval "$(pyenv init --path)"
+eval "$(pyenv init -)"
+eval "$(pyenv virtualenv-init -)"
+```
+> Note: for `fish` shell, add this to `~/.config/fish/config.fish` instead. If you don't know what fish is, ignore this.
```

Comment:
If we're explaining how to configure for fish we should DEFINITELY do so for zsh